.github/workflows/deploy.yaml

```
@@ -9,13 +9,32 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        chain: ["arbitrum-one", "avalanche", "bsc", "celo", "fantom", "fuse", "goerli", "mainnet", "mantle-testnet", "mantle", "matic", "xdai", "optimism", "moonbeam", "tombchain"]
+        chain:
+          [
+            "arbitrum-one",
+            "avalanche",
+            "bsc",
+            "celo",
+            "core",
+            "fantom",
+            "fuse",
+            "goerli",
+            "mainnet",
+            "mantle-testnet",
+            "mantle",
+            "matic",
+            "xdai",
+            "optimism",
+            "moonbeam",
+            "tombchain",
+          ]
     environment:
       name: ${{ matrix.chain }}
       # Use Mantle Foundation graph node for Mantle Testnet
       # Use FuxionX graph node for Mantle
+      # Use Core DAO Foundation graph node for Core
       # Use TheGraph Hosted Service for all other networks
-      url: ${{ matrix.chain == 'mantle-testnet' && 'https://graph.testnet.mantle.xyz/subgraphs/name/requestnetwork/request-payments-mantle-testnet' || matrix.chain == 'mantle' && 'https://graph.fusionx.finance/subgraphs/name/requestnetwork/request-payments-mantle' || format('https://thegraph.com/hosted-service/subgraph/requestnetwork/request-payments-{0}', matrix.chain) }}
+      url: ${{ matrix.chain == 'mantle-testnet' && 'https://graph.testnet.mantle.xyz/subgraphs/name/requestnetwork/request-payments-mantle-testnet' || matrix.chain == 'mantle' && 'https://graph.fusionx.finance/subgraphs/name/requestnetwork/request-payments-mantle' || matrix.chain === 'core' && 'https://thegraph.coredao.org/subgraphs/name/requestnetwork/request-payments-core' || format('https://thegraph.com/hosted-service/subgraph/requestnetwork/request-payments-{0}', matrix.chain) }}
```

This becomes hard to read, apparently, you can use multiline like so:
https://github.com/orgs/community/discussions/25641#discussioncomment-3248574

```suggestion
      url: ${{ matrix.chain == 'mantle-testnet' && 'https://graph.testnet.mantle.xyz/subgraphs/name/requestnetwork/request-payments-mantle-testnet'
               || matrix.chain == 'mantle' && 'https://graph.fusionx.finance/subgraphs/name/requestnetwork/request-payments-mantle'
               || matrix.chain === 'core' && 'https://thegraph.coredao.org/subgraphs/name/requestnetwork/request-payments-core'
               || format('https://thegraph.com/hosted-service/subgraph/requestnetwork/request-payments-{0}', matrix.chain) }}
```

---

Or even better, do like @benjlevesque 's proposed, with a different job